### PR TITLE
Added missing this for system (part 2)

### PR DESCRIPTION
**Pull request by davidpiuva**

### CODE REVIEW

**Comment by davidpiuva** [comment]
@bettinaselig Peer review?

**Comment by bettinaselig** [comment]
Ok.

File: source/system/String.ooc [hunk 1]
```diff
@@ -114,7 +114,7 @@ String: class {
 	capitalize: func -> This {
 		match (this size) {
 			case 0 => this
-			case 1 => toUpper()
+			case 1 => this toUpper()
 			case =>
 				this[0 .. 1] toUpper() + this[1 .. -1]
```
Comment (marcusnaslund) [inline source/system/String.ooc]: Not this PR, but, I just.. what does this line do?

Comment (marcusnaslund) [inline source/system/String.ooc]: Oh, `this[1 .. -1]` eventually becomes `substring(1, size)`, huh. Nvm.

Comment (davidpiuva) [inline source/system/String.ooc]: It looks like it takes upper case of the first character and leaves the rest as it is.

**Comment by marcusnaslund** [comment]
Conflicts, sorry, probably my fault.

**Comment by davidpiuva** [comment]
Rebased.

**Comment by marcusnaslund** [comment]
Still conflicts.

**Comment by davidpiuva** [comment]
No conflicts when I rebased now.

**Comment by marcusnaslund** [comment]
Interesting. I will do this as soon as tests finish to avoid further conflicts.

**Pull request merged**